powerbi/LeadTime(V1)-PowerBI-MQuery.pq

```diff
@@ -0,0 +1,55 @@
+let
+    startDate = "2024-01-20",            // TODO: Replace these dates with the range you want to grab the lead time aggregate for
```

Comment:
You might want to put the date format YYYY-MM-DD in the TODO comment, even though it's obvious from the hardcoded values.